packages/expo-cli/src/commands/prebuild.ts

```diff
@@ -46,5 +49,9 @@ export default function (program: Command) {
     .option('--no-install', 'Skip installing npm packages and CocoaPods.')
     .option('--npm', 'Use npm to install dependencies. (default when Yarn is not installed)')
     .option('-p, --platform [platform]', 'Platforms to sync: ios, android, all. Default: all')
+    .option(
+      '--skip-dependency-update [dependencies]',
+      'Preserves versions of listed packages in package.json (comma separated list)'
```

Comment:
Do we use comma separated anywhere else? Does anyone have a preference towards collected values (`--skip-dependency-update react-native --skip-dependency-update expo-updates`)?

Reply:
> Do we use comma separated anywhere else?

No

> Does anyone have a preference towards collected values

It's the first thing I tried, but something like that is not supported in version of commander we are currently using and based on previous attempts updating commander version is not that easy.